.github/workflows/github-ci.yml

```diff
@@ -272,8 +272,27 @@ jobs:
         path: |
           vlsvextract_DP
           vlsvdiff_DP
+          fluxfunction
         if-no-files-found: error
 
+  test_flixfunction:
```

Comment:
Taipou

Reply:
Fixed (by removing that part oy fthe YaML file)